unified-runtime/source/adapters/level_zero/v2/event.cpp

```diff
@@ -19,6 +19,8 @@
 
 #include "../ur_interface_loader.hpp"
 
+ur_event_generation_t unbatchedQueue = 1;
```

Comment:
This shouldn't be a global object. And the name isn't very descriptive to me. `EVENT_GENERATION_IGNORED` and a comment? I'm not sure. I'd prefer an std::optional, I think it conveys the semantics better ;)

if it's a constant, use `static constexpr UNBATCHED_QUEUE = 1` (in the header if you need it across multiple files).